tests/LiquiditySwapV3.t.sol

```diff
@@ -0,0 +1,410 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {IERC20Detailed} from '@aave/core-v3/contracts/dependencies/openzeppelin/contracts/IERC20Detailed.sol';
+import {IACLManager} from '@aave/core-v3/contracts/interfaces/IACLManager.sol';
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {AaveGovernanceV2} from 'aave-address-book/AaveGovernanceV2.sol';
+import {Errors} from 'aave-address-book/AaveV3.sol';
+import {AaveV3Ethereum, AaveV3EthereumAssets, IPool} from 'aave-address-book/AaveV3Ethereum.sol';
+import {BaseTest} from './utils/BaseTest.sol';
+import {ParaSwapLiquiditySwapAdapterV3} from '../src/contracts/ParaSwapLiquiditySwapAdapterV3.sol';
+import {AugustusRegistry} from '../src/lib/AugustusRegistry.sol';
+import {BaseParaSwapAdapter} from '../src/contracts/BaseParaSwapAdapter.sol';
+import {IParaSwapLiquiditySwapAdapter} from '../src/interfaces/IParaSwapLiquiditySwapAdapter.sol';
+import {IBaseParaSwapAdapter} from '../src/interfaces/IBaseParaSwapAdapter.sol';
+import {stdMath} from 'forge-std/StdMath.sol';
+
+contract LiquiditySwapAdapterV3 is BaseTest {
+  ParaSwapLiquiditySwapAdapterV3 internal liquiditySwapAdapter;
+
+  function setUp() public override {
+    super.setUp();
+    vm.createSelectFork(vm.rpcUrl('mainnet'), 18683100);
+
+    liquiditySwapAdapter = new ParaSwapLiquiditySwapAdapterV3(
+      IPoolAddressesProvider(address(AaveV3Ethereum.POOL_ADDRESSES_PROVIDER)),
+      address(AaveV3Ethereum.POOL),
+      AugustusRegistry.ETHEREUM,
+      AaveGovernanceV2.SHORT_EXECUTOR
+    );
+    vm.startPrank(0x5300A1a15135EA4dc7aD5a167152C01EFc9b192A); //ACL admin
+    IACLManager(address(AaveV3Ethereum.ACL_MANAGER)).addFlashBorrower(address(liquiditySwapAdapter));
```

Comment:
the best thing would be having tests for V3 with and without flashloanFee.
For the sake of keeping tests short, let's have V3 without fee, but let's use the aave address book for the ACL admin. (instead of hard coding address)